packages/web-components/src/components/footer/__stories__/footer.stories.ts

```diff
@@ -60,7 +60,7 @@ export const base = ({ parameters }) => {
             language="${ifNonNull(language)}"
             lang-display="${ifNonNull(langDisplay)}"
             size="${ifNonNull(size)}"
-            languageList="${ifNonNull(langList)}"
+            lang-list="${ifNonNull(JSON.stringify(langList))}"
```

Comment:
You can use `lit-html` property binding syntax to avoid having to de-serialize string: https://lit-html.polymer-project.org/guide/writing-templates#bind-to-properties

```suggestion
            .langList="${ifNonNull(langList)}"
```